### DISCUSSION TITLE
PS3 game control conflicts with 8bitDo Wireless Adapter

### DISCUSSION THREAD

**Discussion by Leomaker** [cathery/sys-con#65] [Troubleshooting]
0.4.2 release.

**Comment by cathery** [comment]
Are you using DS3 over the 8bitdo adapter? What does the adapter recognize the gamepad as? I've seen someone's adapter recognize their DS4 as an xbox 360 controller, to which I can safely say - there's no way to work around that.

**Comment by Leomaker** [comment]
> Are you using DS3 over the 8bitdo adapter?

Yes. The 8bitdo adapter transmits wireless signal to the controller, but the switch recognizes as a wired Pro Controller.

> What does the adapter recognize the gamepad as?

Where do i find this information?

**Comment by cathery** [comment]
> Where do i find this information?

I don't know, I don't have the adapter. But you can find what the sysmodule recognizes your controller as in the log located at `sdmc:/config/sys-con/log.txt`.

My suggestion is to look into the 8bitdo adapter manual or support to see if you can make it fake your controller as a native switch device instead of xbox 360 or whatever it uses right now, because my sysmodule can't possibly detect the difference.

**Comment by Leomaker** [comment]
New information. I updated the 8bitdo adapter firmware (1,26 to 1.28) and now the sysmodules of sys-con no longer cancels the 8bitdo adapter, but if you connect a USB cable with the adapter connected, the switch recognizes as 2 controls at the same time.

**Comment by Leomaker** [comment]
I tested with another control not configured for the 8bitdo adapter and connected it via USB on Switch and was able to play with two controls normally, one wirelessly with 8bitdo and the other plugged into USB via sys-con.

The error mentioned in the previous comment only happens if you connect via USB the control that was paired with the 8bitdo adapter and leave the adapter connected to the switch.

**Comment by mikerm** [comment]
Also would like to report that after updating my 8bitdo adapter to version 1.28, my ds4 works as intended. With also another ds4 plugged in using an usb cable.
